### ISSUE TITLE
Servers not populated. servers.json config not used in 6.19

### ISSUE THREAD

**Issue by pratikyoff** [pgadmin-org/pgadmin4#5751]
**_Please note that security bugs or issues should be reported to security@pgadmin.org._**

**Describe the bug**
servers.json file in /pgadmin4/servers.json is not used and servers are not populated in v6.19. Working as intended with 6.18.
Here is my servers.json file
```
{
    "Servers": {
      "1": {
        "Name": "local-db",
        "Group": "Servers",
        "Host": "dev-postgres",
        "Port": 5432,
        "MaintenanceDB": "postgres",
        "Username": "user",
        "PassFile": "/pgpass",
        "SSLMode": "prefer"
      }
    }
  }
  ```

**To Reproduce**
This is the docker-compose.yml which I am using to run pgadmin4 and postgres
```
version: "3.8"
services:
  db:
    image: postgres
    container_name: dev-postgres
    restart: always
    ports:
      - "5432:5432"
    environment:
      POSTGRES_USER: user
      POSTGRES_PASSWORD: password
    volumes:
      - local_pgdata:/var/lib/postgresql/data

  pgadmin:
    image: dpage/pgadmin4
    container_name: dev-pgadmin4
    restart: always
    ports:
      - "80:80"
    environment:
      PGADMIN_DEFAULT_EMAIL: user@example.com
      PGADMIN_DEFAULT_PASSWORD: password
    volumes:
      - ./.config/pgAdmin/servers.json:/pgadmin4/servers.json
      - pgadmin-data:/var/lib/pgadmin

volumes:
  local_pgdata:
  pgadmin-data:

```

Click on Servers.
It does not get populated with the existing server configuraiton.

**Desktop (please complete the following information):**
 - OS: macOS 13.0.1
 - Version: 6.19
 - Mode: Server
 - Browser Firefox
 - Package type: Container

**Comment by pravesh-sharma** [comment]
Issue resolved. Verified on latest snapshot build https://hub.docker.com/r/dpage/pgadmin4/tags

**Comment by ttfkam** [comment]
Using the official Docker image for 6.20. I have exported servers.json from the web GUI and then attempted to bind servers.json in the new container. It does not load correctly. My volumes section in docker-compose.yml includes as follows:

      - ./docker/pgadmin4/servers.json:/pgadmin4/servers.json

This file is readable and can be referenced from within the container to import the server list manually.

**Comment by till** [comment]
I just wanted to comment or share an observation as this is an icky problem.

At first it did not work for me also. I double checked inside the container too. File was mounted and identical. So I tore down my compose stack completely, pulled the latest tag (6.20), deleted all volumes, etc.. And then it started to work for me again when I add one:

```
ory-dev-pgadmin-1         | ----------
ory-dev-pgadmin-1         | Loading servers with:
ory-dev-pgadmin-1         | User: email@example.org
ory-dev-pgadmin-1         | SQLite pgAdmin config: /var/lib/pgadmin/pgadmin4.db
ory-dev-pgadmin-1         | ----------
```

Then I re-add the second server, deleted everything, restarted and it's also populated now.

But the message I copied from the log wasn't there before as I did this a couple of times trying to figure out if I had something wrong. So it seems like a race maybe? Or maybe I had a typo in the json file? Maybe an error message could be added if that's the case?

Anyway, thanks for publishing pgadmin. :)
